theseus/embodied/kinematics/robot_model.py

```diff
@@ -39,3 +41,27 @@ def forward_kinematics(self, robot_pose: LieGroup) -> torch.Tensor:
 
     def dim(self) -> int:
         return 1
+
+class UrdfRobotModel(RobotModel):
+    def __init__(self, urdf_path: str):
+        self.robot_model = drm.DifferentiableRobotModel(urdf_path)
```

Comment:
Field name and class name (the parent class name might change) being similar might lead to some confusion.